### PR TITLE
config sample improvements

### PR DESCRIPTION
**Pull request by alvarocabanas**
config: update example config with latest recommendations

### CODE REVIEW

File: kafka-config.yml.sample [hunk 1]
```diff
@@ -1,40 +1,69 @@
...
+    # Topic size and offset can also be collected from brokers in the cluster. This operations are intensive and can take a while
+    # to collect for a larger number of topics. It is recommended to only enable this features when collecting topics
+    # selectively, i.e. topic_mode is not "all" or a very permissive regex.
+    # If the fields are omitted they will default to false.
+    COLLECT_TOPIC_SIZE: false
+    COLLECT_TOPIC_OFFSET: false
+
+    METRICS: "true"
   interval: 15s
+  inventory_source: config/kafka
```
Comment (carlossscastro) [inline kafka-config.yml.sample:62]: Should we add this is. a metrics only config?

File: kafka-config.yml.sample [hunk 2]
```diff
@@ -55,85 +87,74 @@ integrations:
...
-    TOPIC_MODE: "all"
-    COLLECT_TOPIC_SIZE: false
+    BOOTSTRAP_BROKER_KAFKA_PROTOCOL: PLAINTEXT
+    # A regex pattern that matches the consumer groups to collect metrics from
+    CONSUMER_GROUP_REGEX: '.*'
   interval: 15s
   labels:
     env: production
     role: kafka
   inventory_source: config/kafka
```
Comment (carlossscastro) [inline kafka-config.yml.sample:152]: Same here

File: kafka-config.yml.sample [hunk 3]
```diff
@@ -55,85 +87,74 @@ integrations:
...
-    # the topics across a number of buckets. The first number is the index of the current instance, the second is the total number of instances the
-    # topics are split across. For example, if you want the topics matched by `topic_regex: 'mytopic.*'` to be split across three instances, one
-    # instance will be configured with `topic_bucket: 1/3`, one with `2/3`, and one with `3/3`
-    TOPIC_BUCKET: '1/3'
+    TOPIC_MODE: "all"
   interval: 15s
   labels:
     env: production
     role: kafka
   inventory_source: config/kafka
```
Comment (carlossscastro) [inline kafka-config.yml.sample:114]: Same here

**Review by carlossscastro — APPROVED** [review]
There's also same unnecessary `inventory_source` in line 183 and 222.
Not a big deal but we might want to remove to be consistent with our documentation

File: kafka-config.yml.sample (hunk 1 above)
Comment (alvarocabanas) [reply kafka-config.yml.sample:62]: yes sure, we can remove it, I will do it before merging

**Pull request merged**